Ensure kernel entropy pool seeding daemon is installed

CHTJonas opened the pull request:
This PR adds the necessary playbook steps to ensure that entropyclient is installed.

@mas90 please could you review and check that I've got the correct steps for your PPA? 😄

CHTJonas committed:
Add steps to install entropyclient

mas90 commented on roles/base/tasks/main.yml:
```diff
@@ -83,4 +83,15 @@
+    name:
+      - entropyclient
+    state: present
+    update_cache: yes
```
Better to update the cache when we change the apt repositories, rather than when we install a package?  (i.e. move that up 6 lines?)

CHTJonas replied:
Looks like `update_cache: yes` is the default for the [apt_repository module](https://docs.ansible.com/ansible/latest/modules/apt_repository_module.html) so this line isn't even necessary. I'll remove.

CHTJonas committed:
Remove superfluous configuration

The apt_repository module updates the apt cache by default when changing state.
